README.md

```diff
@@ -1,38 +1,41 @@
 # FortiGate Autoscale
+
 A collection of **Node.js** modules and cloud-specific templates which support autoscale functionality for groups of FortiGate-VM instances on various cloud platforms.
 
 This project contains the code and templates for the **AliCloud Autoscale** deployment.
```

Comment:
```suggestion
This project contains the code and templates for the **FortiGate Autoscale for AliCloud** deployment.
For autoscale on **GCP** see the [fortigate-autoscale-gcp](https://github.com/fortinet/fortigate-autoscale-gcp/) repository.
For autoscale on **Amazon AWS** and **Microsoft Azure** see the [fortigate-autoscale](https://github.com/fortinet/fortigate-autoscale/) repository.

Reply:
templates for ~the~ **FortiGate Autoscale for AliCloud** deployment. ?